plugins/module_utils/job.py

```diff
@@ -36,7 +36,7 @@
 )
 
 
-def job_output(job_id=None, owner=None, job_name=None, dd_name=None, duration=0, timeout=0, start_time=timer()):
+def job_output(job_id=None, owner=None, job_name=None, dd_name=None, dd_scan=True, duration=0, timeout=0, start_time=timer()):
```

Comment:
Since dd_scan is added as a parameter in job_output function please also update the docstring of the function

Reply:
Thank you for seeing this.  I've made this change.

Reply:
Overall, I'm not sure how we can test that data is NOT being pulled, which is why I didn't add a test.

Reply:
agree